### PR TITLE
Resolve VCSWP-20372 (TranscribeUtterance not generating as a PerclCommand)

### PR DESCRIPTION
**Pull request by amahadaya**
Closes VCSWP-20372

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -3269,6 +3263,78 @@
...
+              "prompts": [
+                {
+                  "Play": {
+                    "file": "http://dplutil01.vail/~user/prompts/please-record-greeting.wav"
+                  }
+                }
+              ]
+            }
+          }
+        }
```
Comment (ngarg-vail) [inline openapi.json:3337]: Why is this appearing as a diff in this way? shouldn't there just be an `x-percl-command` update? Also it shouldn't be removing properties unless that was an intended change.

File: test/test_transcribe_utterance.py [hunk 2]
```diff
@@ -24,56 +24,42 @@ class TestTranscribeUtterance(unittest.TestCase):
...
         self.model.play_beep = False
         assert self.model.get("play_beep") == False
-        
-         
 
-     
     def test_record(self):
         """Test TranscribeUtterance.record"""
-        self.model.record = TranscribeUtteranceRecord(save_recording=False, max_length_sec=1, rcrd_termination_silence_time_ms=1)
-        assert isinstance(self.model.record, TranscribeUtteranceRecord)
```
Comment (ngarg-vail) [inline test/test_transcribe_utterance.py:48]: Why is this removed?

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: freeclimb/model/transcribe_utterance.py [hunk 3]
```diff
@@ -100,6 +155,7 @@ def openapi_types():
     @cached_property
     def discriminator():
         return None
+        return None
```
Comment (ngarg-vail) [inline freeclimb/model/transcribe_utterance.py]: Returning twice. this is wrong.

File: openapi.json (hunk 1 above)
Comment (amahadaya) [reply openapi.json:3337]: This is an intended change, Logan had removed the x-max-lengths properties and this openapi.json file did not have this change, so once autogenerated, it applied the openapi changes from the latest branch to this file

File: test/test_transcribe_utterance.py (hunk 2 above)
Comment (amahadaya) [reply test/test_transcribe_utterance.py:48]: I believe latest commit should add the test back in

File: openapi.json [hunk 4]
```diff
@@ -3269,6 +3263,78 @@
...
+              "record": {
+                "maxLengthSec": 60,
+                "saveRecording": true,
+                "rcrdTerminationSilenceTimeMs": 2000
+              },
+              "playBeep": true,
+              "prompts": [
+                {
+                  "Play": {
+                    "file": "http://dplutil01.vail/~user/prompts/please-record-greeting.wav"
```
Comment (ngarg-vail) [inline openapi.json]: we need to remove this from the spec *asap*

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

File: freeclimb/model/transcribe_utterance.py (hunk 3 above)
Comment (ngarg-vail) [reply freeclimb/model/transcribe_utterance.py]: Needs to be resolved before approval

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,12 +9,15 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="4.5.2"></a>
```
Comment (ngarg-vail) [inline CHANGELOG.md]: remove

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**